src/cryptojwt/key_issuer.py

```diff
@@ -297,7 +297,7 @@ def get(self, key_use, key_type="", kid=None, alg="", **kwargs):
             else:
                 _bkeys = bundle.keys()
             for key in _bkeys:
-                if key.inactive_since and key_use != "sig":
+                if key.inactive_since and key_use != "ver":
```

Comment:
where is `"ver"` as a value for `"use"` defined? AFAIK, `rfc7517` only defines the following

>   Values defined by this specification are:
>
>   o  "sig" (signature)
>   o  "enc" (encryption)

Reply:
Yes that true, but reading the comment about "key_use" seems that it has been extended. The basic value are stored in "use". Reading the comment below, this seemed the correct thing to do.

Reply:
For this I'm not sure what's the right value to use. @rohe should chip in ;) 
We can also do `key_use noy in ["sig", "ver"]` to cover both cases, but I'm not sure this is what we need.

Reply:
The right value to use is "sig". "ver" is a leftover from way back when it was used for keys that was only to be used for verification and never for signing. Today one should do that using `keyops` rather than `use`.